dashboard/test/models/script_test.rb

```diff
@@ -1696,6 +1696,36 @@ class SummarizeVisibleAfterScriptTests < ActiveSupport::TestCase
     assert_equal('CSP Test', assignable_info[:category])
   end
 
+  # This test checks that all categories that may show up in the UI have
+  # translation strings.
+  test 'all visible categories have translations' do
```

Comment:
I tried to put this test in test_script_constants.rb but it looked like I18n wasn't available there.  Does putting this test here seem ok?